overlord/devicestate/devicectx.go

```diff
@@ -30,6 +32,23 @@ func DeviceCtx(st *state.State, task *state.Task, providedDeviceCtx snapstate.De
 	if providedDeviceCtx != nil {
 		return providedDeviceCtx, nil
 	}
+
+	// see if we have a remodel in progress
+	if task != nil {
+		var modelass string
+		if err := task.Change().Get("new-model", &modelass); err == nil {
+			ass, err := asserts.Decode([]byte(modelass))
```

Comment:
Nitpick: s/ass/assertion/ or s/ass/assert/ (the latter seems to be used the most in existing code); easier on the eyes 😊 ; same for modelass (sounds even more...uhm)..

Reply:
*cough* good points - especially about modelAs - will fix.